packages/react/scripts/name.d.ts

```diff
@@ -1,5 +1,5 @@
 export declare function generateScopedName(
   name: string,
   filename: string,
-  css: string,
+  css?: string,
```

Comment:
This seems to be unused? Can we remove it, so the type is correct?

Reply:
Should we not keep it? Its "almost" correct according to official type: https://github.com/madyankin/postcss-modules/blob/325f0b33f1b746eae7aa827504a5efd0949022ef/index.d.ts#L3